### PR TITLE
Bump crates versions

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: _data/crates.json [hunk 1]
```diff
@@ -3,36 +3,36 @@
     "section": "Core",
 
     "name": "cairo-rs",
-    "max_version": "0.15.1",
+    "max_version": "0.16.1",
```
Comment (GuillaumeGomez) [inline _data/crates.json:6]: `0.16.1` directly?

File: _data/crates.json [hunk 2]
```diff
@@ -3,36 +3,36 @@
...
   {
     "name": "gio",
-    "max_version": "0.15.1",
+    "max_version": "0.16.0",
     "repo": "gtk-rs-core"
   },
   {
     "name": "glib",
-    "max_version": "0.15.1",
+    "max_version": "0.16.1",
```
Comment (GuillaumeGomez) [inline _data/crates.json:16]: `0.16.1` directly?

File: _data/crates.json (hunk 1 above)
Comment (bilelmoussaoui) [reply _data/crates.json:6]: we had a new fix release already

File: _data/crates.json (hunk 2 above)
Comment (bilelmoussaoui) [reply _data/crates.json:16]: same as cairo-rs

**Comment by GuillaumeGomez** [comment]
Then all good, thanks!

**Pull request merged**